src/01-dinosaur-facts.js

```diff
@@ -44,7 +66,16 @@ function getLongestDinosaur(dinosaurs) {}
  *  getDinosaurDescription(dinosaurs, "incorrect-id");
  *  //> "A dinosaur with an ID of 'incorrect-id' cannot be found."
  */
-function getDinosaurDescription(dinosaurs, id) {}
+function getDinosaurDescription(dinosaurs, id) {
+  const foundDinosaur = dinosaurs.find((dinosaur) => dinosaur.dinosaurId === id);
+  
+  if (foundDinosaur) {
+    return `${foundDinosaur.name} (${foundDinosaur.pronunciation})\n${foundDinosaur.info} It lived in the ${foundDinosaur.period} period, over ${foundDinosaur.mya[0]} million years ago.`;
```

Comment:
great spot for a helper function here to shorten this long chain of string interpolation